typing/subst.ml

```diff
@@ -174,7 +201,11 @@ let rec typexp copy_scope s ty =
     let has_fixed_row =
       not (is_Tconstr ty) && is_constr_row ~allow_ident:false tm in
     (* Make a stub *)
-    let ty' = if s.for_saving then newpersty (Tvar None) else newgenvar () in
+    let layout = Type_layout.any in
```

Comment:
Regarding this comment and the one just below it:  Basically my impression is you're describing an improvement that could make the code in this (complicated!) function clearer - by distinguishing stubs from variables - but which is kind of orthogonal to the layouts patch (because if the stubs aren't filled in, we already have a bug).  Since I don't think we can really be making the situation worse here, I'd suggest putting that on some pile of nice-to-have type systems improvements, but not changing anything here now.  But perhaps I misunderstand this code or your comment.